news/2018-06-02-mappers-choice-awards-2017-results.md

```diff
@@ -0,0 +1,345 @@
+
+*written by [Tofu1222](https://osu.ppy.sh/u/6089608)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/maxus.jpg)](https://osu.ppy.sh/u/4335785)
+
+You can never find a more welcoming and active modder in the osu!mania mapping community than **[Maxus](https://osu.ppy.sh/u/4335785)**. His high activity in modding and BN work and his proficiency in mapping and guiding new mappers definitely paved the way for him to win the osu!mania modder category. 
+
+*written by [Tofu1222](https://osu.ppy.sh/u/6089608)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/protastic.jpg)](https://osu.ppy.sh/u/6712747)
+
+When we talked about the amount of high quality consistency in the mod contribution, **[Protastic101](https://osu.ppy.sh/u/6712747)** is undoubtedly one that we must mentioned. It's very occassional seeing someone in their BN era to be really passionate in keep helping new mapper on how they improve and patiently explain in a long length what the mapper must fix to achieve map in a higher quality. She is one of the pioneer that makes the ranked section much more accessible for the new mapper and certainly her contribution really deserves a few words than just an amazement that makes a huge sign in the mania history.
+
+*written by [Maxus](https://osu.ppy.sh/u/4335785)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/tofu.jpg)](https://osu.ppy.sh/u/6089608)
+
+**[Tofu1222](https://osu.ppy.sh/u/6089608)** growth in the mapping community is certainly fast when we looked on his first map ranked, and the amount of map he has ranked since. He displays an enormous passion on digging emphasis on various genres of music, and also even started to map in other modes to learn more from them. A clear winner for the Most Promising category.
+
+*written by [Maxus](https://osu.ppy.sh/u/4335785)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/evening.jpg)](https://osu.ppy.sh/u/2193881)
+
+**[Evening](](https://osu.ppy.sh/u/2193881))**'s mapping journey already give a huge insight of what he already accomplished not only in the amount of maps that people already support him, but also on how he brings new concepts to mania not only in ranking section but also to what the mania editor actually capable of. 
+
+A prime example is **[Planet Shaper](https://osu.ppy.sh/s/376293)**, a singularity map that proves the hidden capability of what the mania mapping can do with impressive red line manipulation and the multitude of ideas derived from it. With almost 2 months of work put into it, it goes to show how much passion that he pours into his maps. The last thing is how he continuously helps people from either new or old mappers by handling various level of mapping, and even helping out other modes with timing and storyboards. What he brings and contribute into osu mania is certainly underrated by many, and he really deserves the Most Influential title.
```

Comment:
a singularity map -> uhh marathon or some wording like that
with impressive -> with
The last thing is how he -> In addition to his mapping, he
people from either new or old mappers by handling various level of mapping -> both old and new mappers with all kinds of mapping styles
even helping -> helps
brings and contribute into -> contributes to
the Most Influential title -> the title of Most Influential.